modules/ding_react/ding_react.module

```diff
@@ -419,6 +420,16 @@ function ding_react_cover_service_url() {
   return variable_get('ding_react_cover_service_url', DING_REACT_COVER_SERVICE_URL);
 }
 
+/**
+ * Returns the url to the instance of the related materials service to use.
+ *
+ * @return string
+ *   Url to related materials service instance.
+ */
+function ding_react_related_materiols_service_url() {
```

Comment:
is the "materiols" spelling error on purpose?

Reply:
Sadly no, fixed.